src/solvers/dgsem_p4est/containers_parallel.jl

```diff
@@ -66,27 +66,126 @@ function init_mpi_interfaces(mesh::ParallelP4estMesh, equations, basis, elements
 end
 
 function init_mpi_interfaces!(mpi_interfaces, mesh::ParallelP4estMesh)
-  init_surfaces!(nothing, nothing, nothing, mpi_interfaces, mesh)
+  init_surfaces!(nothing, nothing, nothing, mpi_interfaces, nothing, mesh)
 
   return mpi_interfaces
 end
 
-# Overload init! function for boundaries and regular interfaces since it must call the appropriate
-# init_surfaces! function for parallel p4est meshes
+
+# Container data structure (structure-of-arrays style) for DG L2 mortars
+#
+# Similar to `P4estMortarContainer`. The field `neighbor_ids` has been split up into
+# `local_element_ids` and `local_element_positions` to describe the ids and positions of the locally
```

Comment:
OK, then maybe leave it as it is for now (to not make this PR grow in size), create an issue, and fix it in a subsequent PR? I personally like "neighbor" better then "element", since at least to me it is more descriptive.

Reply:
Created issue #1086 to track this.